cmdx.py

```diff
@@ -3861,13 +3948,144 @@ def _python_to_mod(value, plug, mod):
     return True
 
 
+class MetaComponent(type):
+
+    @withTiming()
+    def __call__(cls, mdagpath, mobject, exists=True, modifier=None):
+
+        if mobject.hasFn(om.MFn.kMeshVertComponent):
```

Comment:
Can this not be a method of Component directly? Why the metaclass?

Reply:
Yeah i was unsure about it. I did this to copy how it was implemented on Node. But this was lazy, I think it can be rewritten as you propose